includes/dashboard/showSelectedGroup.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+/* 
+ * this is to show the selected group of the users, selected by the admin
+ */
+session_start();
+
+if( !( isset( $_SESSION['Username'] ) && isset($_SESSION['Name']) && $_SESSION['Username'] === 'admin' ) )
+{
+	echo "To access this page you need to login as Admin. Please log in first.";
+	die();
+}
+
+
+
+if ( ! isset($_POST['groupId']) )
+{
+	echo "Sorry, there is some problem";
+	die ();
+}
+
+$groupID = rtrim($_POST['groupId'], ';');
+
+
+require_once "../../LocalSettings.php";
+require_once "../Globals.php";
+
+
+
+$sqlConn = new mysqli( $eorgDBserver , $eorgDBuser , $eorgDBpasswd , $eorgDBname );
+
+if( $sqlConn->connect_errno ) 
+{
+	echo "Internal Server Error, Contact Administrator.";
+	die();
+}
+
+
+
+echo "<h2><br><br><br>The available users are :<br><br></h2>";
+
+
+
+
+$i = 1;
+
+$html = <<<HTML
+<table>
+
+<tr>
+<th>Sr. No.</th>
+<th>User ID</th>
+<th>Name</th>
+<th>Sex</th>
+<th>Designation(s)</th>
+</tr>
+HTML;
+
+echo $html;
+
+
+$qry = "SELECT * FROM $loginDB WHERE $Desig REGEXP \"^$groupID\"" ;
+$stmt = $sqlConn->prepare ( $qry );
```

Comment:
__[code_sniffer]__
* Space before opening parenthesis of function call prohibited
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found

<!-- Comment by [SideCI](https://sideci.com) -->